src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,44 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File readable = new File(fromFileName);
+        File writable = new File(toFileName);
+        StringBuilder builder = new StringBuilder();
+        int supplyValue = 0;
+        int buyValue = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(readable));
+            String value = reader.readLine();
+            while (value != null) {
+                if (value.contains("supply")) {
+                    supplyValue += Integer.parseInt(value.substring(value.indexOf(',') + 1));
+                }
+                if (value.contains("buy")) {
+                    buyValue += Integer.parseInt(value.substring(value.indexOf(',') + 1));
```

Comment:
1 - see checklist - point about substring()
2 - look through checklist and try to split logic into several small methods
3 - use constants for magic numbers and regexes